lib/eChronicler/journal_entries.ex

```diff
@@ -8,10 +8,20 @@ defmodule EChronicler.JournalEntries do
     Repo.all(JournalEntry)
   end
 
+  def reverse_chron_list_journal_entries() do
+    Repo.all(JournalEntry)
+    |> Enum.sort(&(&1.inserted_at > &2.inserted_at))
+  end
+
   def create_journal_entry(attrs \\ %{}) do
     %JournalEntry{}
     |> JournalEntry.changeset(attrs)
     |> Repo.insert()
   end
 
+  def truncate_journal_entry(entry) do
```

Comment:
Kind of unclear whether this acts on a `%JournalEntry{}` struct or the journal entry text.

Reply:
I changed the name of the function to be clearer